packages/components/src/card/test/body.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
```

Comment:
I guess it's not written anywhere, but we've been trying to slowly move away from "enzyme". I think we settled on just using the tools provided by the React team to avoid having tests blocking us from upgrade React in future updates...

cc @gziolo

Reply:
Yes, see the related issue and discussion there:
https://github.com/WordPress/gutenberg/issues/17249

There was an effort in the past when @nerrad had to refactor over 20 test files to make them work without `Enzyme` because they were behind with updating their API by a few months.

Reply:
Oh boy! @gziolo , I didn't know about this thread! But I went over (slash ranted) about testing difficulty with Enzyme on my live coding session 😂 .

@youknowriad Is there an example of component tests I should follow?
I'm more than happy to refactor/remove the tests if it's inline with the direction and future of `@wordpress/component` testing

Reply:
The usage of `update` to refresh components is very tricky and it was introduced as a workaround to the fix their previous architectural decisions :(

Reply:
> Is there an example of component tests I should follow?
> I'm more than happy to refactor/remove the tests if it's inline with the direction and future of `@wordpress/component` testing

We probably don't have too many unit tests covering UI components, which use something different than `Enzyme`. I guess it's fine to keep your tests for the time being and figure out our strategy in the linked issue (#17249).

Reply:
Sounds good! Thank you!